specification/ai/Azure.AI.Projects/main.tsp

```diff
@@ -2,16 +2,6 @@ import "@typespec/http";
 import "@typespec/rest";
 import "@typespec/versioning";
 import "@azure-tools/typespec-azure-core";
-import "./agents/routes.tsp";
```

Comment:
Shouldn't you also be deleting all these Agents files?

Reply:
We have removed the agents files, however, projects still hold connections, evaluations and telemetry endpoints.

Reply:
The files were oved to Azure.AI.Agents.